platform/darwin/src/MGLRendererConfiguration.m

```diff
@@ -63,4 +64,20 @@ - (BOOL)perSourceCollisionsWithInfoDictionaryObject:(nullable id)infoDictionaryO
     return NO;
 }
 
+- (MGLGlyphsRasterizationMode)glyphsRasterizationMode {
+    id infoDictionaryObject = [NSBundle.mainBundle objectForInfoDictionaryKey:MGLGlyphsRasterizationModeKey];
+    return [self glyphsRasterizationModeWithInfoDictionaryObject:infoDictionaryObject];
+}
+
+- (MGLGlyphsRasterizationMode)glyphsRasterizationModeWithInfoDictionaryObject:(id)infoDictionaryObject {
+    if (!infoDictionaryObject || ![infoDictionaryObject isKindOfClass:[NSString class]]) {
+        return MGLNoGlyphsRasterizedLocally;
```

Comment:
I think it should return something that represents 'unset' value, `nil` cc/ @julianrex?

With this PR, if user don't set `MGLIdeographicFontFamilyName` and `MGLGlyphsRasterizationMode`, `glyphsRasterizationModeWithInfoDictionaryObject` would return `MGLNoGlyphsRasterizedLocally`, thus, we will not render CJK characters locally. That will be a breaking change.

Reply:
Excellent point. Rather than `nil` (which would make this return type a bit weird), what if we add `MGLGlyphsRasterizationModeNone`?